src/kubernetes.h

```diff
@@ -152,6 +189,18 @@ class KubernetesReader {
   // A memoized map from an encoded owner reference to the owner object.
   mutable std::map<std::string, json::value> owners_;
 
+  // Unique identifier of a service in a cluster, based on the namespace name
```

Comment:
How about: `// ServiceKey is a pair of the namespace name and the service name that uniquely identifies a service in a cluster.`?

Reply:
done